docs/rules/no-namespace.md

```diff
@@ -5,6 +5,12 @@ Enforce a convention of not using namespace (a.k.a. "wildcard" `*`) imports.
 +(fixable) The `--fix` option on the [command line] automatically fixes problems reported by this rule, provided that the namespace object is only used for direct member access, e.g. `namespace.a`.
 The `--fix` functionality for this rule requires ESLint 5 or newer.
 
+### Options
+
+This rule supports the following options:
+
+- `ignoreFromPattern`: regular expression for modules that should be ignored by the rule.
```

Comment:
I'd probably just name this "ignore", and let it be an array of glob strings (the description needs updating)